quarkus/runtime/src/main/java/org/keycloak/QuarkusKeycloakApplication.java

```diff
@@ -66,4 +68,10 @@ private void forceEntityManagerInitialization() {
         // when first creating an entity manager
         entityManagerFactory.get().createEntityManager().close();
     }
+
+    private void initializeAdditionalFeatures() {
+        KeycloakSession session = sessionFactory.create();
+        // Client Policies
+        session.clientPolicy().setupClientPoliciesOnKeycloakApp("/keycloak-default-client-profiles.json", "/keycloak-default-client-policies.json");
```

Comment:
It seems that in quarkus, the call to `session.clientPolicy().setupClientPoliciesOnKeycloakApp` is done at quite different step than on the Wildfly distribution? For example in the Wildfly ( KeycloakApplication.startup ), it is done before sending PostMigrationEvent and setup of scheduled tasks. In the quarkus, it is done after that. I suggest to align both distributions to avoid some unexpected side-effects. Maybe would be slightly better to call `initializeAdditionalFeatures();` from the `initializeKeycloakSessionFactory()` ? Maybe after call `instance.init()` (line 62) is a good place?

Reply:
@mposolda I understood the point. As you suggested, I will call `session.clientPolicy().setupClientPoliciesOnKeycloakApp` just after the line 62 `instance.init()` of `QuarkusKeycloakApplication.initializeKeycloakSessionFactory` in order to call it before sending PostMigrationEvent and setup of scheduled tasks the same as in wildfly distribution.